src/core/lv_obj_property.c

```diff
@@ -165,23 +207,37 @@ static lv_result_t obj_property(lv_obj_t * obj, lv_prop_id_t id, lv_property_t *
             if(!set) value->id = prop->id;
 
             switch(LV_PROPERTY_ID_TYPE(prop->id)) {
-                case LV_PROPERTY_TYPE_INT:
-                    if(set)((lv_property_set_int_t)(prop->setter))(obj, value->num);
-                    else value->num = ((lv_property_get_int_t)(prop->getter))(obj);
-                    break;
+                case LV_PROPERTY_TYPE_INT: {
+                        if(set)((lv_property_set_int_t)(prop->setter))(obj, value->num);
+                        else value->num = ((lv_property_get_int_t)(prop->getter))(obj);
+                        break;
+                    }
+                case LV_PROPERTY_TYPE_PRECISE: {
+                        if(set)((lv_property_set_precise_t)(prop->setter))(obj, value->precise);
+                        else value->precise = ((lv_property_get_precise_t)(prop->getter))(obj);
+                        break;
+                    }
+                case LV_PROPERTY_TYPE_COLOR: {
+                        if(set)((lv_property_set_color_t)prop->setter)(obj, value->color);
+                        else value->color = ((lv_property_get_color_t)(prop->getter))(obj);
+                        break;
+                    }
+                case LV_PROPERTY_TYPE_POINT: {
+                        lv_point_t * point = &value->point;
+                        if(set)((lv_property_set_pointer_t)(prop->setter))(obj, point);
+                        else *point = ((lv_property_get_point_t)(prop->getter))(obj);
```

Comment:
Should `lv_property_set_pointer_t` be `lv_property_set_point_t`? You declared `lv_property_set_point_t` at the top.